src/cljs/proton/layers/lang/latex/core.cljs

```diff
@@ -0,0 +1,35 @@
+(ns proton.layers.lang.latex.core
+  (:use [proton.layers.base :only [init-layer! get-initial-config get-keybindings get-packages get-keymaps]]))
+
+(defmethod get-initial-config :lang/latex []
+  ["proton.lang.latex.use-latex-plus" false])
+
+; Define default packages
+(def packages
+  (atom
+    [:latexer
+     :language-latex
+     :autocomplete-bibtex
+     :pdf-view]))
+
+(defmethod init-layer! :lang/latex [_ config]
+  (let [config-map (into (hash-map) config)]
+    (if (config-map "proton.lang.latex.use-latex-plus")
```

Comment:
> proton.lang.latex.use-latex-plus

yeah we should definitely try to get the new config system (https://github.com/dvcrn/proton/issues/22) out anytime soon. I think all these config strings might get confusing 

Looking good to me